modules/update-upgrading-cli.adoc

```diff
@@ -75,19 +61,22 @@ VERSION IMAGE
 4.10.12 quay.io/openshift-release-dev/ocp-release@sha256:f77f4f75c1e1a4ddd0a0355f298a834db3473fd9ca473235013e9419d1df16db
 4.10.13 quay.io/openshift-release-dev/ocp-release@sha256:4f516616baed3cf84585e753359f7ef2153ae139c2e80e0191902fbd073c4143
 ----
+[NOTE]
+*For details and information on how to perform an `EUS-to-EUS` channel upgrade, please refer to the 
+_Preparing to perform an EUS-to-EUS upgrade page_, listed in the Additional resources section.
```

Comment:
Note formatting:

[NOTE]
====
For details and information on how to perform an `EUS-to-EUS` channel upgrade, please refer to the 
_Preparing to perform an EUS-to-EUS upgrade page_, listed in the Additional resources section.
====

Reply:
s/_Preparing to perform an EUS-to-EUS upgrade page_/_Preparing to perform an EUS-to-EUS upgrade_ page

Reply:
Thank you! Adjusted both :)